client/Makefile

```diff
@@ -21,51 +21,33 @@ WEBPACK_CONFIG ?= configuration/webpack/webpack.config.prod.js
 build:
 	npm run build $(WEBPACK_CONFIG)
 
-# Formatting code
-
-.PHONY: lint
-lint:
-	npm run lint
-
 # Development convenience methods
 .PHONY: start-frontend
 start-frontend:
 	node server/development.js
 
-.PHONY: e2e
-e2e:
-	APP_PORT=${CXG_SERVER_PORT} \
-		npm run e2e -- \
-		--verbose false \
-
-.PHONY: e2e-annotations
-e2e-annotations:
-	APP_PORT=${CXG_SERVER_PORT} \
-	npm run e2e-annotations -- \
-		--verbose false \
-
+# start an instance of cellxgene and run the end-to-end tests
 .PHONY: smoke-test
 smoke-test:
 	start_server_and_test \
 		'CXG_OPTIONS="--disable-annotations" $(MAKE) start-server' \
 		$(CXG_SERVER_PORT) \
-		'$(MAKE) e2e'
-
+		'CXG_URL_BASE="https://localhost:$(CXG_SERVER_PORT)" npm run e2e -- --verbose false'
 
+# start an instance of cellxgene and run the end-to-end annotations tests
 .PHONY: smoke-test-annotations
 smoke-test-annotations:
 	$(eval TMP_DIR := $(shell  mktemp -d /tmp/cellxgene_XXXXXX))
 	cp $(ANNOTATIONS) $(TMP_DIR)/ && \
 	start_server_and_test \
 		'CXG_OPTIONS="--annotations-file $(TMP_DIR)/$(ANNOTATIONS_FILENAME)" $(MAKE) start-server' \
 		$(CXG_SERVER_PORT) \
-		'$(MAKE) e2e-annotations'
+		'CXG_URL_BASE="https://localhost:$(CXG_SERVER_PORT)" npm run e2e-annotations -- --verbose false'
```

Comment:
ooh nice collapsing the logic here 🔥 👏 !!

Reply:
@tihuan just trying to further deduplicate things here. Tests are failing though. Trying to figure out why.